### DISCUSSION TITLE
Automation: how to fill relationship field

### DISCUSSION THREAD

**Discussion by gvlx** [Budibase/budibase#3340] [Help]
In an automation, triggered when a new row is created, I want to fill a relationship field automatically.

How can this be done?

In a screen, you can create actions which can map relationship fields:
![imagem](https://user-images.githubusercontent.com/67311/141426147-58b0f4d4-a309-48b2-99a4-d3f85da73fc6.png)

In an automation, you must choose the value:
![imagem](https://user-images.githubusercontent.com/67311/141426527-8baed96b-6a6a-4387-95a5-53334c9f1de1.png)

Maybe there is a way to create a Javascript automation step which sets the values and updates a row?

**Comment by mjashanks** [comment]
I have logged this as an issue: https://github.com/Budibase/budibase/issues/3344
